index.html

```diff
@@ -63,6 +63,236 @@ <h1>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="grid-container">
+          <img
+            class="image-fluid"
+            src="images/works/image_3.png"
+            alt="background_image"
+          />
+          <div class="text">
+            <span class="tonic">Tonic</span>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="works_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="grid-container">
+          <img
+            class="image-fluid space-sm"
+            src="images/works/image_4.png"
+            alt="background_image"
+          />
+          <div class="text">
+            <span class="multi-post"
+              >Multi-Post<br />
+              Stories</span
+            >
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="works_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="vertical-space"></div>
+      </section>
+
+      <section class="about">
+        <h1>
+          About<br />Myself<br />
+          you!
```

Comment:
- [ ] [OPTIONAL] I think this text is not necessary. How about if you remove it?

Reply:
@BertrandConxy I have removed that text.
![screencapture-127-0-0-1-5501-2022-07-04-12_43_42 (2)](https://user-images.githubusercontent.com/29442846/177139325-052f2a9d-c503-46a9-9bb0-672dcd2e6417.png)